build/helper/metadata_filters.py

```diff
@@ -158,11 +169,12 @@
         'reordered_for_default_values': False,
         'skip_repeated_capability_parameter': False,
         'skip_non_enum_parameter': False,
+        'skip_numpy_parameters': False,
         'skip_all_except_numpy_parameters': False,
-        'mechanism': 'fixed, passed-in, len, python-code',  # any but ivi-dance
+        'mechanism': 'any',
```

Comment:
The generated output looks good, so it's probably correct, but why was it okay to set "any" as the mechanism for these _OUTPUT_PARAMETERS objects?

Reply:
I changed it in [this commit](https://github.com/ni/nimi-python/pull/1822/commits/f3accbd72cf01b8a8d74e368d7b58a7a1a7b5754) to explicitly check for ivi-dance parameters in `_mock_helpers.py` instead of using this filtering mechanism. I don't remember my exact thought process, but I think it was probably for code clarity: in `_mock_helpers.py` where we use it, we already iterate over the ivi-dance parameters separately, and IMO it's clearer to do both checks (ivi-dance vs. non-ivi-dance) in one place.